src/auth/auth.guard.ts

```
@@ -0,0 +1,46 @@
+      //Step 2: Decrypt the Token
+      const jwtDecryptedToken = await jose.jwtDecrypt(token, hash);
+      
+      if (!jwtDecryptedToken.payload.jwtSignedToken) {
+        throw new Error("jwtSignedToken not found in decrypted payload");
+      }
+
+      //Step 3: Verify the Signed JWT
+      const jwtSignedToken = String(jwtDecryptedToken.payload.jwtSignedToken);
+    
+      //Fix Signing Key
+      const jwtSigninKey = new TextEncoder().encode(process.env.JWT_SIGNIN_PRIVATE_KEY);
```

ESLint

[error] Replace `process.env.JWT_SIGNIN_PRIVATE_KEY` with `⏎········process.env.JWT_SIGNIN_PRIVATE_KEY,⏎······`

(prettier/prettier)